src/services/bulkstatus.service.js

```diff
@@ -40,7 +40,13 @@ async function checkBulkStatus(req, res) {
   }
 
   logger.debug(`Retrieved the following bulkStatus entry for client: ${clientId}. ${JSON.stringify(bulkStatus)}`);
-  if (bulkStatus.status === 'In Progress') {
+  if (
+    bulkStatus.status === 'In Progress' ||
+    (bulkStatus.status === 'In Progress' &&
```

Comment:
Not sure what the second part of this `||` is supposed to do since it will only be true if the first part of the `||` is true?

Reply:
I have no clue why this was added. I'm going to check if this can be removed.